kubetest/aksengine.go

```diff
@@ -88,6 +88,16 @@ var (
 	cloudProviderAzureVersion = getImageVersion(util.K8sSigs("cloud-provider-azure"))
 	imageRegistry             = os.Getenv("REGISTRY")
 	k8sNodeTarballDir         = util.K8s("kubernetes", "_output", "release-tars") // contains custom-built kubelet and kubectl
+
+	// kubemark scale tests
```

Comment:
need split the PR into two PRs:

1) kubetest changes
2) job config changes after the kubetest changes are included in the image